tueplots/figsize.py

```diff
@@ -26,6 +26,11 @@ def cvpr(*, column="half", nrows=1):
 
 
 # Single-column formats
+def beamer(*, nrows=1):
+    """beamer figure size"""
```

Comment:
Nice! 

Is this a full-width beamer image? Shall we eventually add `column="full"/"half"` like in the ICML sizes? 

```suggestion
def beamer(*, nrows=1):
    """Beamer figure size for `aspectratio=169`."""
```
Maybe we can extend this with beamer(*, nrows=1, column="half", aspectratio="169") in the future. (But no need for now).

Reply:
Yes, I agree, let's add a half column (and the general case of fractional width)